Algorithms.cuh

```diff
@@ -1,11 +1,30 @@
 #pragma once
 
+#include "../../../../../../usr/lib/gcc/x86_64-linux-gnu/6/include/stdint-gcc.h"
```

Comment:
Bad automatic import?
```suggestion
```

Reply:
Probably, I didn't mean to write that line.

Reply:
Ok I commit the change